compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -325,6 +338,11 @@ public void visit(BLangPackage pkgNode, AnalyzerData data) {
                 continue;
             }
 
```

Comment:
Aren't you reanalyzing type definitions again here?

Reply:
No, only the ones that weren't analyzed.